test/unit/cmap/connection_monitoring_and_pooling.spec.test.js

```diff
@@ -0,0 +1,281 @@
+'use strict';
```

Comment:
Should this be in `test/integration` instead of `test/unit` along with the connection string spec test? They are the only 2 in `test/unit` and even though they are closer to unit tests than integration tests with have other similar ones like uri options that's moved to `test/integration`. Maybe better to not have these 2 stragglers?